MAUI/Scheduler/accessibility.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+title: Accessibility in .NET MAUI SfScheduler control | Syncfusion
+description: Learn here about Accessibility support with the Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements and more.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Accessibility Support in .NET MAUI Scheduler (SfScheduler)
+
+## Keyboard navigation
+
+The `SfScheduler` supports selecting the scheduler date time, appointments, and view navigation using keyboard interaction in WinUI, MAC Catalyst, and iOS platforms.
```

Comment:
The `SfScheduler` supports selecting the scheduler date time, appointments, and view navigation using keyboard interactions in WinUI, MAC Catalyst, and iOS platforms.